src/pybind/mgr/cephadm/module.py

```diff
@@ -1123,6 +1123,9 @@ def _run_cephadm(self,
             if not no_fsid:
                 final_args += ['--fsid', self._cluster_fsid]
 
+            if self.container_init:
+                final_args += ['--container-init']
```

Comment:
a few lines above we have 

```
1116            if not self.container_init:
1117                final_args += ['--no-container-init']
```
Are those lines correct in octous?

Reply:
Yeah -- 442904b adds --no-container-init to cephadm, and I think it's fine to keep both these flags here, as it makes it the cephadm mgr module explicitly specify one or the other, regardless of what the internal cephadm default might be

Reply:
Would it make sense to combine the two `if` statements into one like so:

```
if self.container_init:
    final_args += ['--container-init']
else:
    final_args += ['--no-container-init']
```

?

Reply:
I'd like to, but due to the way those parameters are parsed, I don't think I can. `--no-container-init` is accepted at the top level (e.g.: `cephadm --no-container-init [command] ...`) whereas `--container-init` is only accepted inside individual commands (`cephadm adopt --container-init`, `cephadm bootstrap --container-init`, `cephadm deploy --container-init`), so I think it has to remain structured as it is now, i.e.:

```
            if not self.container_init:
                final_args += ['--no-container-init']

            # subcommand
            final_args.append(command)

            # subcommand args
            if not no_fsid:
                final_args += ['--fsid', self._cluster_fsid]

            if self.container_init:
                final_args += ['--container-init']
```

Reply:
yeah, `--container-init` was only accepted for the bootstrap, deploy and adopt commands, whereas the `--no-container-init` param is global to all cephadm commands ...